ingest/rules/nextclade.smk

```diff
@@ -48,10 +43,6 @@ rule run_nextclade:
             --output-fasta {output.alignment} \
           &> {log:q}
         """
-        #     --output-translations {params.translations}
-
-        # zip -rj {output.translations} results/translations
-        # """
 
 
 rule join_metadata_and_nextclade:
```

Comment:
_non-blocking_

Since you're making changes here, consider updating the `join_metadata_and_nextclade` rule to use `augur rename` and `augur merge` (see https://github.com/nextstrain/pathogen-repo-guide/pull/67)

Reply:
Broke this suggestion out into #21 — I'd like to focus on getting this landed, and this won't change anything about the final dataset, so it feels okay to defer it for the moment.